ldk/javascript/examples/react-loop/package.json

```diff
@@ -0,0 +1,76 @@
+{
+  "name": "react-loop",
+  "version": "0.0.1",
+  "description": "",
+  "main": "src/index.js",
+  "scripts": {
+    "build": "webpack --entry ./src/index.js --config ./node_modules/@oliveai/ldk/dist/webpack/config.js",
```

Comment:
You have a webpack config for the Loop but the build script is pointing to the LDK webpack config. You might want to just run `webpack` and I think it will handle the rest.

Reply:
Changed to just be `"build": "webpack"`